Zend/zend_compile.h

```diff
@@ -285,6 +285,13 @@ typedef struct _zend_oparray_context {
 /* stored in opcache (may be partially)                   |     |     |     */
 #define ZEND_ACC_CACHED                  (1 << 22) /*  X  |     |     |     */
 /*                                                        |     |     |     */
+/* temporary flag used during delayed variance checks     |     |     |     */
+#define ZEND_ACC_CACHABLE                (1 << 23) /*  X  |     |     |     */
```

Comment:
Also here, `CACHEABLE` rather than CACHABLE (sorry, mostly scanning the patch, as I lack background to evaluate its technical merits)

Reply:
Thanks. Fixed.